ksqldb-engine/src/main/java/io/confluent/ksql/engine/EngineExecutor.java

```diff
@@ -347,28 +369,128 @@ TransientQueryMetadata executeTransientQuery(
     );
   }
 
+  @SuppressFBWarnings(value = "NP_NULL_PARAM_DEREF_NONVIRTUAL")
```

Comment:
I had to add this because calling the `new QueryPlan(...)` with null values was failing.